src/Maker/MakeCrud.php

```diff
@@ -41,10 +43,22 @@ final class MakeCrud extends AbstractMaker
 
     private $formTypeRenderer;
 
-    public function __construct(DoctrineHelper $doctrineHelper, FormTypeRenderer $formTypeRenderer)
+    /** @var Inflector|null */
+    private $inflector;
+
+    public function __construct(DoctrineHelper $doctrineHelper, FormTypeRenderer $formTypeRenderer, Inflector $inflector = null)
     {
         $this->doctrineHelper = $doctrineHelper;
         $this->formTypeRenderer = $formTypeRenderer;
+        $this->inflector = $inflector;
+
+        if (null === $inflector) {
+            @trigger_error(sprintf('"%s" will require a value for the $inflector argument in 2.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
This is a bit tricky. When testing with lowest deps (so `doctrine/inflector` 1.0), MakerBundle's config will not pass this argument, and so it triggers a deprecation. In other words: if you use MakeBundle with `doctrine/inflector` 1.0, you would see this deprecation no matter what (unless you upgraded `doctrine/inflector` to `2.0`.

One easy fix would be to not add a new argument: just `InflectorFactory::create()->build();` like we do with `Validator`. Obviously, that's not normally something we would do, but is there any practical downside?

Reply:
No practical downside. If you were to support multiple languages, you’d want to inject it, but given how we hardcode to English we may as well push this change back, at least until we require 1.4 and always have the new API.